code/datums/gamemode/objectives/divergentclone.dm

```diff
@@ -0,0 +1,63 @@
+/datum/objective/freeform/divergentclone_neutral
+    explanation_text = "Convince the world that you are the real <person>, or at least as real as the other copy."
```

Comment:
<person>
What did he mean by this?

Reply:
This is mostly just a placeholder/sanity thing. The actual text gets set in PostAppend.